curriculum/challenges/english/25-front-end-development/quiz-html-accessibility/66ed9026f45ce3ece4053ebb.md

```diff
@@ -17,439 +17,446 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder distractor 1
+WAI-ARIA is a programming language specifically designed for creating animated web graphics
 
 ---
 
-Placeholder distractor 2
+WAI-ARIA is a set of guidelines for making websites more visually appealing and colorful
 
 ---
 
-Placeholder distractor 3
+WAI-ARIA is a tool for optimizing website loading speeds and server performance.
 
 #### --answer--
 
-Placeholder answer
+WAI-ARIA is a set of standardized practices that ensure online content can be viewed by users with disabilities.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of the `hidden` attribute?
 
 #### --distractors--
 
-Placeholder distractor 1
+The function of the `hidden` attribute is to shrink the focused elements
 
 ---
 
-Placeholder distractor 2
+The function of the `hidden` attribute is to add tabs
 
 ---
 
-Placeholder distractor 3
+The function of the `hidden` attribute is to hide an HTML form
 
 #### --answer--
 
-Placeholder answer
+The function of the `hidden` attribute is to hide certain submitted information from view
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the function of the `tabindex` attribute?
 
 #### --distractors--
 
-Placeholder distractor 1
+The function of the `tabindex` attribute is to change the color of focused elements
 
 ---
 
-Placeholder distractor 2
+The function of the `tabindex` attribute is to shrink the focused elements
 
 ---
 
-Placeholder distractor 3
+The function of the `tabindex` attribute is to add tabs 
 
 #### --answer--
 
-Placeholder answer
+The function of the `tabindex` attribute is to control the order of keyboard navigation on a webpage
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What are the best methods for creating accessible data tables? 
 
 #### --distractors--
 
-Placeholder distractor 1
+Add a caption by using the `<paragraph>` tag
 
 ---
 
-Placeholder distractor 2
+Use the `aria-label` attribute to make sure that the data cells associate with the proper header
 
 ---
 
-Placeholder distractor 3
+Use absolute sizing to make sure that the data cells are not too large
 
 #### --answer--
 
-Placeholder answer
+Designate row and/or column headers by using `<td>` and`<th>` tags
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the difference between the attributes `aria-label` and `aria-labelledby`?
 
 #### --distractors--
 
-Placeholder distractor 1
+The `aria-label`and `aria-labelledby` attributes serve the same purpose.
 
 ---
 
-Placeholder distractor 2
+The `aria-label` attribute is used to give an color to its label while the `aria-labelledby` attribute is used to give height to its label.
 
 ---
 
-Placeholder distractor 3
+The `aria-label` and `aria-labelledby` attributes keeps labels undefined. 
 
 #### --answer--
 
-Placeholder answer
+The `aria-labelledby` attribute is used to give an element a label while the `aria-label` attribute is used to give an element its label. 
```

Comment:
I'm sorry, but I don't understand this answer. Both of these attributes are used to give an element an accessible name (or label if you prefer). The only difference is that `aria-labelledby` allows you to do that by referencing an existing element on the page while `aria-label` allows you to define the name in the attribute itself.